core/vm/tosca_ct_integration.go

```diff
@@ -0,0 +1,137 @@
+package vm
+
+import (
+	"errors"
+	"strings"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/holiman/uint256"
+)
+
+// Error
+var ErrStopToken = errStopToken
```

Comment:
Since we now also have a state enumeration in the `InterpreterState` I think this error is no longer needed to be exported.